tests/build_/test_arrangements.py

```diff
@@ -428,3 +429,32 @@ def test_add_filesystem_in_worker(self, osbs, base_image, scratch,
             assert 'koji_hub' in args
             assert args['repos'] == params['yum_repourls']
             assert args['from_task_id'] == params['filesystem_koji_task_id']
+
+    @pytest.mark.parametrize(('scratch', 'base_image', 'expect_plugin'), [  # noqa:F811
+        (True, 'koji/image-build', False),
+        (True, 'foo', False),
+        (False, 'koji/image-build', False),
+        (False, 'foo', True),
+    ])
+    def test_koji_parent_in_orchestrator(self, osbs, base_image, scratch,
+                                         expect_plugin):
+        additional_params = {
+            'base_image': base_image,
+            'yum_repourls': ['https://example.com/my.repo'],
```

Comment:
No, it isn't. I thought for some reason that yum_repourls was required for detecting it's a custom base image build. Removed.